<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="robot_name"/>
    <arg name="init_pose"/>
    <group ns="$(arg robot_name)">
        <param name="tf_prefix" value="$(arg robot_name)_tf" />
        <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find beacon_gazebo_plugin)/urdf/autonet_v1.xacro'" />


        <node name="robot_spawn__$(arg robot_name)" pkg="gazebo_ros" type="spawn_model" output="screen"
              args="$(arg init_pose) -urdf -param robot_description -model $(arg robot_name)" />
        <!--0 -0.1 0.2" rpy="0 0.5 0  -->
        <node name="robot_state_publisher__$(arg robot_name)" pkg="robot_state_publisher" type="robot_state_publisher" />
    </group>
</launch>